src/main/java/com/ericsson/ei/rules/MatchIdRulesHandler.java

```diff
@@ -67,7 +73,10 @@ public String replaceIdInRules(String matchIdString, String id) {
         if (matchIdString.contains(replacementMarker)) {
             return matchIdString.replace(replacementMarker, id);
         } else {
-            return null;
+            String errorMessage = String.format("MatchIdRules: %s does not contain the rules.replacement.marker: %s", matchIdString, replacementMarker);
```

Comment:
Will this not cause it to throw exception and fail each time rules.replacement.marker is not set?

If rules.replacement.marker is not set should this not be able to work if ID is already provided?

Reply:
I guess you could just replace the the marker in MatchIdRules no matter what replacement marker says.

Reply:
This is handled differently now.